cmd/common.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	sp "cloud.google.com/go/spanner"
+	"github.com/cloudspannerecosystem/harbourbridge/common/utils"
+	"github.com/cloudspannerecosystem/harbourbridge/conversion"
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/profiles"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/writer"
+	"github.com/cloudspannerecosystem/harbourbridge/streaming"
+)
+
+func startDatastream(ctx context.Context, sourceProfile profiles.SourceProfile, targetProfile profiles.TargetProfile) (streaming.StreamingCfg, error) {
+	streamingCfg := streaming.StreamingCfg{}
+	var err error
+	if sourceProfile.Ty == profiles.SourceProfileTypeStreaming {
```

Comment:
if it is not a streaming profile, we don't do anything ?
This seems like it is silently ignoring if it is not a streaming profile, which may lead to confusion.

Reply:
We have a few options to make it clearer
1. Check for streaming profile outside of this method
2. Add a log return if it is not a streaming profile